docker-compose.yml

```diff
@@ -27,13 +26,15 @@ services:
       - 3005:3000
     links:
       - store-db
+      - admin-db
```

Comment:
store-api doesn't need this link to admin-db?

Reply:
Fixed